### PR TITLE
Rewrites of Debugging, Workspaces, Modes

### PR DESCRIPTION
**Pull request by NWeber-SL**

### CODE REVIEW

File: using-ocular/getting-started/workspaces.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+The workspace can contain multiple CPGs. When you [create a CPG](working-with-cpg.md), it is automatically loaded into the workspace. Existing CPGs can also be [loaded into a workspace, and removed or deleted from a workspace](working-with-cpg.md). By default, all operations are executed on the CPG that was last loaded into the workspace; this is the active CPG.
+
+The active, last loaded CPG is accessible through the `cpg` variable; all CPGs loaded in the workspace are accessible using the `cpgs` variable.
+
+## Exploring the Workspace
+
+Issuing the command `workspace` shows a workspace entry for the newly created CPG.
+
+```
+workspace
```
Comment (vlad-alexandru-ionescu) [inline using-ocular/getting-started/workspaces.md]:
```suggestion
ocular> workspace
```

File: using-ocular/getting-started/workspaces.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+The workspace can contain multiple CPGs. When you [create a CPG](working-with-cpg.md), it is automatically loaded into the workspace. Existing CPGs can also be [loaded into a workspace, and removed or deleted from a workspace](working-with-cpg.md). By default, all operations are executed on the CPG that was last loaded into the workspace; this is the active CPG.
+
+The active, last loaded CPG is accessible through the `cpg` variable; all CPGs loaded in the workspace are accessible using the `cpgs` variable.
+
+## Exploring the Workspace
+
+Issuing the command `workspace` shows a workspace entry for the newly created CPG.
+
+```
```
Comment (vlad-alexandru-ionescu) [inline using-ocular/getting-started/workspaces.md]:
```suggestion
```scala
```

File: using-ocular/getting-started/workspaces.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+The CPG that was loaded last is accessible using the variable `cpg`.
+
+## Managing Overlays
+
+You manage overlays by showing all overlay creators, loading a CPG only with a `semanticcpg` overlay and adding an overlay. 
+
+To show all available overlay creators, use
+
+`overlays`
```
Comment (vlad-alexandru-ionescu) [inline using-ocular/getting-started/workspaces.md]: Should use

    ```scala
    ocular> overlays
    ```

File: using-ocular/getting-started/workspaces.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+where `filename` is the target application.
+
+To add an overlay, use 
+
+* `addOverlay("overlay")`. 
+
+where `overlay` is the name of the overlay you want to added, for example `tagging`.
+
+## Identifying Application Code and Dependencies
```
Comment (vlad-alexandru-ionescu) [inline using-ocular/getting-started/workspaces.md]: Everything below this is not related to workspaces. It should be moved to a separate documentation page.

File: using-ocular/getting-started/workspaces.md (hunk 1 above)
Comment (NWeber-SL) [reply using-ocular/getting-started/workspaces.md]: Workspaces file replaced by Working with CPGs in PR #105

File: using-ocular/getting-started/workspaces.md (hunk 2 above)
Comment (NWeber-SL) [reply using-ocular/getting-started/workspaces.md]: Workspaces file replaced by Working with CPGs in PR #105

File: using-ocular/getting-started/workspaces.md (hunk 3 above)
Comment (NWeber-SL) [reply using-ocular/getting-started/workspaces.md]: Workspaces file replaced by Working with CPGs in PR #105

File: using-ocular/getting-started/workspaces.md (hunk 4 above)
Comment (NWeber-SL) [reply using-ocular/getting-started/workspaces.md]: Yes, it was pulled out in Working with CPGs
